js/5etools-main.js

```diff
@@ -1503,6 +1503,8 @@ const betteR205etoolsMain = function () {
 					else if ('special' in item) {
 						iname = item.special;
 					}
+					
+					if (item.containsValue) startingGold = item.containsValue/100;
```

Comment:
this is spaghetti--return a starting gold total from the `parseItems` method, instead of setting an "external" variable here

also, there's a bug here if you supply multiple items with `containsValue`; only the last will be used. This `.map` should e.g. total the values for the item list it's iterating over

Reply:
`parseItems` is called within other blocks so returning a value in that scope will not exist on the line that we want to assign it to a trait. We will still need a more global/external variable. At least that is what i think. Am i wrong?

Reply:
> We will still need a more global/external variable

Sure, keep the `let startingGold` at the outer level, but don't modify it from within the function. Add the return value from the function to it, instead, for example.

Essentially: if I were to refactor `parseItems` out to a nicer non-nested place, I shouldn't have to change anything besides the references to the `parseItems` function itself. The function should have its inputs, and its outputs, and not change any "external" state in between.

Reply:
ok thank you ... i think it is what it need to be now!